src/components/avatars/OcAvatarItem.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+  <span
+    class="oc-avatar-item"
+    :style="{ backgroundColor: backgroundColor, '--icon-color': iconColor }"
+    :aria-label="accessibleLabel === '' ? null : accessibleLabel"
+    :aria-hidden="accessibleLabel === '' ? 'true' : null"
+    :focusable="accessibleLabel === '' ? 'false' : null"
+    :role="accessibleLabel === '' ? null : 'img'"
+    :data-test-link-name="name"
+  >
+    <oc-icon v-if="hasIcon" :name="icon" size="small"></oc-icon>
+  </span>
+</template>
+<script>
+export default {
+  name: "OcAvatarItem",
+  status: "ready",
+  release: "2.1.0",
+
+  props: {
+    /**
+     * Name of the public link used as an accessible label
+     */
+    name: {
+      type: String,
+      required: true,
+    },
+    /**
+     * Icon that should be used for the avatar
+     */
+    icon: {
+      type: String,
+      required: false,
+      default: null,
+    },
+    /**
+     * Color that should be used for the icon
+     */
+    iconColor: {
+      type: String,
+      required: false,
+      default: "#ffffff",
+    },
+    /**
+     * Background color that should be used for the avatar. If empty
+     * a random color will be picked
+     */
+    background: {
+      type: String,
+      required: false,
+      default: "var(--oc-color-swatch-brand-default)",
+    },
+    /**
+     * Accessibility label used as alt. Use only in case the avatar is used alone.
+     * In case the avatar is used next to username or display name leave empty.
+     * If not specified, avatar will get `aria-hidden="true"`.
+     **/
+    accessibleLabel: {
+      type: String,
+      required: false,
+      default: "",
+    },
+  },
+
+  computed: {
+    hasIcon() {
+      return this.icon !== null
+    },
+    backgroundColor() {
+      return this.background ? this.background : this.pickBackgroundColor
```

Comment:
I know this existed before, but this could be simplified as `return this.background || this.pickBackgroundColor`.